05-services.md

```diff
@@ -850,12 +850,23 @@ external_links:
 `extra_hosts` adds hostname mappings to the container network interface configuration (`/etc/hosts` for Linux).
 
 ### Short syntax
-Short syntax uses plain strings in a list. Values must set hostname and IP address for additional hosts in the form of `HOSTNAME:IP`.
+Short syntax uses plain strings in a list. Values must set hostname and IP address for additional hosts in the form of `HOSTNAME=IP`.
+
+```yml
+extra_hosts:
+  - "somehost=162.242.195.82"
+  - "otherhost=50.31.209.229"
+  - "myhostv6=::1"
+```
+
+IPv6 addresses can be enclosed in square brackets and, for backward
```

Comment:
is this "backward compatibility" as use of `:` is deprecated in docker/cli and user is warned (then https://github.com/compose-spec/compose-go/pull/499 should also do), or is this just an alternate (better?) syntax ? In such case, both should be documented, and maybe `=` recommended to prevent ambiguities

Reply:
Ok, got it - thank you. So, instead of ...

> IPv6 addresses can be enclosed in square brackets and, for backward compatibility, the separator `:` can be used. For example:
> 
> extra_hosts:
>   - "somehost:162.242.195.82"
>   - "otherhost:50.31.209.229"
>   - "myhostv6:[::1]"

How about this? ...

> IPv6 addresses can be enclosed in square brackets, for example:
> 
> extra_hosts:
>   - "myhost=[::1]"
>
> The separator `=` is preferred, but `:` can also be used. For example:
> 
> extra_hosts:
>   - "somehost:162.242.195.82"
>   - "myhostv6:::1"